jenkins.advisories.yaml

```
@@ -2,6 +2,13 @@ package:
   name: jenkins
 
 advisories:
+  CVE-2016-1000027:
+    - timestamp: 2023-07-21T14:56:46.359543-04:00
+      status: under_investigation
+    - timestamp: 2023-07-21T15:11:25.939734-04:00
+      status: not_affected
+      justification: vulnerable_code_not_in_execute_path
+
```

Your conclusion sounds great to me! Could you add an "impact statement" (the `impact:` field) with a brief version of what's in your PR description? The benefit is that the explanation can be carried forward to consumers of the advisory data.

For an example of what this looks like elsewhere, check out [this](https://github.com/wolfi-dev/advisories/blame/9a6f6002bce46c3df2fe5c44cf6dfdd500db5484/gnupg.advisories.yaml#L24)!

```suggestion
    - timestamp: 2023-07-21T15:11:25.939734-04:00
      status: not_affected
      justification: vulnerable_code_not_in_execute_path
      impact: [insert explanation prose here!]
```